src/main/decompInitMod.F90

```diff
@@ -342,15 +379,33 @@ subroutine decompInit_lnd_gindex_global_allocate( bounds, ier )
             call endrun(msg="allocation error for gindex_global", file=sourcefile, line=__LINE__)
             return
          end if
+         ! TODO: Remove the data, and only use the subroutine to calculate when needed
```

Comment:
Remove this section and remove storing these bits of data, and only calculate them in endrun.